packages/feedback/src/core/createMainStyles.ts

```diff
@@ -11,54 +11,26 @@ function getThemedCssVariables(theme: FeedbackInternalOptions['themeLight']): st
   --border: ${theme.border};
   --border-radius: ${theme.borderRadius};
   --box-shadow: ${theme.boxShadow};
-
-  --submit-background: ${theme.submitBackground};
-  --submit-background-hover: ${theme.submitBackgroundHover};
-  --submit-border: ${theme.submitBorder};
-  --submit-outline-focus: ${theme.submitOutlineFocus};
-  --submit-foreground: ${theme.submitForeground};
-  --submit-foreground-hover: ${theme.submitForegroundHover};
-
-  --cancel-background: ${theme.cancelBackground};
-  --cancel-background-hover: ${theme.cancelBackgroundHover};
-  --cancel-border: ${theme.cancelBorder};
-  --cancel-outline-focus: ${theme.cancelOutlineFocus};
-  --cancel-foreground: ${theme.cancelForeground};
-  --cancel-foreground-hover: ${theme.cancelForegroundHover};
-
-  --input-background: ${theme.inputBackground};
-  --input-foreground: ${theme.inputForeground};
-  --input-border: ${theme.inputBorder};
-  --input-outline-focus: ${theme.inputOutlineFocus};
-
-  --form-border-radius: ${theme.formBorderRadius};
-  --form-content-border-radius: ${theme.formContentBorderRadius};
   `;
 }
 
 /**
  * Creates <style> element for widget actor (button that opens the dialog)
  */
-export function createMainStyles(
-  colorScheme: 'system' | 'dark' | 'light',
-  themes: Pick<FeedbackInternalOptions, 'themeLight' | 'themeDark'>,
-): HTMLStyleElement {
+export function createMainStyles({ colorScheme, themeDark, themeLight }: FeedbackInternalOptions): HTMLStyleElement {
   const style = DOCUMENT.createElement('style');
   style.textContent = `
 :host {
-  --z-index: ${themes.themeLight.zIndex};
-  --font-family: ${themes.themeLight.fontFamily};
-  --font-size: ${themes.themeLight.fontSize};
+  --z-index: ${themeLight.zIndex};
+  --font-family: ${themeLight.fontFamily};
+  --font-size: ${themeLight.fontSize};
 
   font-family: var(--font-family);
   font-size: var(--font-size);
 
   --page-margin: 16px;
   --actor-inset: auto var(--page-margin) var(--page-margin) auto;
 
-  --dialog-inset: auto var(--page-margin) var(--page-margin) auto;
-  --dialog-padding: 24px;
-
```

Comment:
these and a bunch of stuff above are in Dialog.css.ts now, along with all the `:host { ... }` stuff to wrap it.